README.rst

```diff
@@ -34,12 +34,12 @@ specifically,
    provides elastic net and group lasso regularization, but only for
    linear and logistic regression.
 
-**Pyglmnet** is a response to this fragmentation. Here are some
-highlights.
+**Pyglmnet** is a response to this fragmentation. It runs on Python 3.5+,
+and here are some of the highlights.
```

Comment:
This might conflict a bit with this PR: https://github.com/glm-tools/pyglmnet/pull/313/files. But I guess I could rebase and adjust